ansible/playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml

```diff
@@ -11,3 +11,7 @@ Nagios_Plugins: Disabled
 ### Default JDK ###
 bootjdk: hotspot
 heapsize: normal
+
+## Nagios Server Details
+Nagios_Monitoring: Enabled
```

Comment:
where this variable is used?

Reply:
comment, it isnt, it was copied and pasted in error, so I've removed it